pythonforandroid/recipes/libzmq/__init__.py

```diff
@@ -69,6 +72,8 @@ def get_recipe_env(self, arch):
         env['CXXFLAGS'] += ' -lgnustl_shared'
         env['LDFLAGS'] += ' -L{}/sources/cxx-stl/gnu-libstdc++/{}/libs/{}'.format(
             self.ctx.ndk_dir, self.ctx.toolchain_version, arch)
+        env['CXXFLAGS'] += ' --sysroot={}/platforms/android-{}/arch-arm'.format(
+            self.ctx.ndk_dir,self.ctx.ndk_api)
```

Comment:
Thanks for the PR, could you simply fix the linting error:
```
pythonforandroid/recipes/libzmq/__init__.py:76:29: E231 missing whitespace after ','
```
For you info, you can run `tox` to verify it upfront.

Reply:
Ok done, I don't know this tool, I'm Android developer but I like python

Reply:
Nice to have some contributions from an Android developer :smile: